clustering/ejb/infinispan/src/main/java/org/wildfly/clustering/ejb/infinispan/InfinispanBeanManager.java

```diff
@@ -286,7 +286,6 @@ Node locatePrimaryOwner(I id) {
             InfinispanEjbLogger.ROOT_LOGGER.debugf("Could not find bean %s", id);
             return null;
         }
-        this.schedule(bean);
```

Comment:
There needs to be be some action taken at the point to prevent concurrent expiration, in particular, when the bean is schedule to be expired by a different member than the one currently performing the invocation.

Reply:
I added prepareRescheduling method below, btw, all methods and names are candidates for renaming if the name doesnt sound right, it is hard to get it right sometimes when you're not a native speaker ;-)